pkg/new_storage/general.go

```diff
@@ -105,17 +105,14 @@ func (bd *BackupDestination) BackupList() ([]Backup, error) {
 			})
 			return nil
 		}
-		mf, err := bd.StatFile(path.Join(o.Name(), "metadata.json"))
+		metadataFilePath := ""
```

Comment:
Did you run `./tests/integration/run.sh` to check your changes for other types of remote storage?

Reply:
also, look like recursive walking will list lot of files which is not folder with backup name

Reply:
> also, look like recursive walking will list lot of files which is not folder with backup name

I think that is ok. Without "recursive=true", we can not get some information about some backup.